src/components/MessageList.vue

```diff
@@ -338,6 +339,12 @@ export default {
 
             return message.isHighlight;
         },
+        userStatus(user) {
+            if (state.getActiveNetwork().ircClient.network.cap.isEnabled('account-notify')) {
```

Comment:
Since this would a rather hot function, moving this `state.getActiveNetwork().ircClient.network.cap.isEnabled('account-notify')` into it's own computed property in this component would cache the result for some performance gains.

Secondly, why is this reliant on account-notify? Should it be away-notify?